rst_files/lln_clt.rst

```diff
@@ -217,82 +202,58 @@ The dots represent the underlying observations :math:`X_i` for :math:`i = 1, \ld
 
 In each of the three cases, convergence of :math:`\bar X_n` to :math:`\mu` occurs as predicted
 
-.. code-block:: julia 
-    :class: test 
+Setup
+------------------
 
-    using Test 
+.. literalinclude:: /_static/includes/deps.jl
 
 .. code-block:: julia
+    :class: test
 
-    using Plots, Distributions, LaTeXStrings, Random, Statistics
-    gr(fmt=:png)
-    
-    n = 100
-    Random.seed!(42)  # reproducible results
-
-    
-    distributions = Dict("student's t with 10 degrees of freedom" => TDist(10), # arbitrary collection of distributions
-        "β(2, 2)" => Beta(2.0, 2.0),
-        "lognormal LN(0, 1/2)" => LogNormal(0.5),
-        "γ(5, 1/2)" => Gamma(5.0, 2.0),
-        "poisson(4)" => Poisson(4),
-        "exponential with lambda = 1" => Exponential(1))
-
-    num_plots = 3
-    dist_data = zeros(num_plots, n)
-    sample_means = []
-    dist_means = []
-    titles = []
-    for i ∈ 1:num_plots
-        dist_names = collect(keys(distributions))
-        
-        # choose a randomly selected distribution
-        name = dist_names[rand(1:length(dist_names))]
-        dist = pop!(distributions, name)
-
-        # == Generate n draws from the distribution == #
-        data = rand(dist, n)
-
-        # == Compute sample mean at each n == #
-        sample_mean = zeros(n)
-        for j ∈ 1:n
-            sample_mean[j] = mean(data[1:j])
-        end
+    using Test
 
-        m = mean(dist)
+.. code-block:: julia
+
+    using StatPlots, Distributions, Random, Statistics
+    gr(fmt = :png, size = (900, 500))
 
-        dist_data[i, :] = data'
-        push!(sample_means, sample_mean)
-        push!(dist_means, m * ones(n))
-        push!(titles, name)
+.. code-block:: julia
 
+    function ksl(distribution, n = 100)
+        title = replace(replace(string(distribution), r"\{.*\}" => ""), r".=" => "")
```

Comment:
This is a bit clever, could we try something with `distribution.type` which returns a `Core.TypeName`. Maybe this can be stringified?